compiler/rustc_error_messages/locales/en-US/passes.ftl

```diff
@@ -534,3 +531,30 @@ passes_no_main_function =
     .consider_adding_main_at_crate = consider adding a `main` function at the crate level
     .teach_note = If you don't know the basics of Rust, you can go look to the Rust Book to get started: https://doc.rust-lang.org/book/
     .non_function_main = non-function item at `crate::main` is found
+
+passes_duplicate_lang_item =
+    {$message ->
+        *[duplicate] found duplicate lang item `{$lang_item_name}`
+        [duplicate_in_crate] duplicate lang item in crate `{$crate_name}`: `{$lang_item_name}`.
```

Comment:
I think recently we've been trying to avoid dispatching on strings in the Fluent code and instead have multiple Fluent messages and an enum in the Rust side or something like that.

Reply:
I took a stab at eliminating fluent string dispatch in https://github.com/rust-lang/rust/pull/102110/commits/2c0b53a42dddaee03c796e5dce5cf1ffa1b98f33 -- is that what you had in mind?